NEWS.md

```diff
@@ -1,6 +1,13 @@
 
 # renv 0.18.0  (UNRELEASED)
 
+* **Breaking Change**: `renv` now includes platform prefix in library paths
```

Comment:
```suggestion
* **Breaking Change**: `renv` now includes platform prefix in cache paths
```
?

Reply:
In fact, the change effects both:

```
> renv:::renv_paths_library()
[1] "/home/kevin/.cache/R/renv/library/renv-3022e2f8/linux-ubuntu-jammy/R-4.1/aarch64-unknown-linux-gnu"
> renv:::renv_paths_cache()
[1] "/tmp/Rtmpoiba2M/renv-tempfile-2ac82649a830d/cache/v5/linux-ubuntu-jammy/R-4.1/aarch64-unknown-linux-gnu"
```

Reply:
Oh so it's not just the cache that will break, but every existing project will need to re-install all packages? I think that's a pretty strong argument for adding some help to automate the transition.